packages/itwin/reports-config-widget/src/test/ReportAction.test.tsx

```diff
@@ -5,87 +5,29 @@
 import "@testing-library/jest-dom";
-import type {
-  IModelConnection,
-  SelectionSet,
-  SelectionSetEvent,
-} from "@itwin/core-frontend";
-import { NoRenderApp } from "@itwin/core-frontend";
 import { ReportsConfigWidget } from "../ReportsConfigWidget";
 import { setupServer } from "msw/node";
 import {
   render,
   screen,
-  TestUtils,
   waitForElementToBeRemoved,
 } from "./test-utils";
-import ReportAction from "../widget/components/ReportAction";
+import { ReportAction } from "../widget/components/ReportAction";
 import userEvent from "@testing-library/user-event";
 import { rest } from "msw";
 import type { Report } from "@itwin/insights-client";
-import * as moq from "typemoq";
 import { REPORTS_CONFIG_BASE_URL } from "../widget/ReportsConfigUiProvider";
-import type {
-  SelectionManager,
-  SelectionScopesManager,
-} from "@itwin/presentation-frontend";
-import {
-  Presentation,
-  SelectionChangeEvent,
-} from "@itwin/presentation-frontend";
-import type { BeEvent } from "@itwin/core-bentley";
 import { EmptyLocalization } from "@itwin/core-common";
 
-const mockITwinId = faker.datatype.uuid();
-const mockIModelId = faker.datatype.uuid();
-
-const connectionMock = moq.Mock.ofType<IModelConnection>();
-const selectionManagerMock = moq.Mock.ofType<SelectionManager>();
-const selectionScopesManagerMock = moq.Mock.ofType<SelectionScopesManager>();
-
-jest.mock("../widget/components/ReportMappings", () => ({
-  ReportMappings: () => "MockReportMappings",
-}));
-
-jest.mock("@itwin/appui-react", () => ({
-  ...jest.requireActual("@itwin/appui-react"),
-  useActiveIModelConnection: () => connectionMock.object,
-}));
-
 const server = setupServer();
 
 beforeAll(async () => {
-  await NoRenderApp.startup({localization: new EmptyLocalization()});
-  await Presentation.initialize();
-  const selectionSet = moq.Mock.ofType<SelectionSet>();
-  const onChanged = moq.Mock.ofType<BeEvent<(ev: SelectionSetEvent) => void>>();
-  selectionSet.setup((x) => x.elements).returns(() => new Set([]));
-  selectionSet.setup((x) => x.onChanged).returns(() => onChanged.object);
-  connectionMock
-    .setup((x) => x.selectionSet)
-    .returns(() => selectionSet.object);
-  connectionMock.setup((x) => x.iModelId).returns(() => mockIModelId);
-  connectionMock.setup((x) => x.iTwinId).returns(() => mockITwinId);
-
-  selectionManagerMock
-    .setup((x) => x.selectionChange)
-    .returns(() => new SelectionChangeEvent());
-
-  selectionScopesManagerMock
-    .setup(async (x) => x.getSelectionScopes(connectionMock.object))
-    .returns(async () => []);
-  selectionManagerMock
-    .setup((x) => x.scopes)
-    .returns(() => selectionScopesManagerMock.object);
-
-  Presentation.setSelectionManager(selectionManagerMock.object);
-  await TestUtils.initializeUiFramework(connectionMock.object);
-  await ReportsConfigWidget.initialize();
+  const localization = new EmptyLocalization();
+  await ReportsConfigWidget.initialize(localization);
   server.listen();
```

Comment:
What is the server used for? It should not be required if all clients are mocked.

Reply:
Only one test required it. I quickly removed it.

Reply:
What do you mean by "removed it"? It's still there.

Reply:
Gone.